tasks/clean/remove_valid_filtered_master_domains.yml

```diff
@@ -0,0 +1,16 @@
+- block:
+    - name: Fetch active/maintenance/detached storage domain for remove
+      ovirt_storage_domains_facts:
+          pattern:  name={{ storage['dr_' + dr_source_map + '_name'] }} and {{ dr_active_domain_search }} or name={{ storage['dr_' + dr_source_map + '_name'] }} and {{ dr_maintenance_domain_search }} or name={{ storage['dr_' + dr_source_map + '_name'] }} and {{ dr_unattached_domain_search }}
+          auth: "{{ ovirt_auth }}"
+
+    - include_tasks: tasks/clean/remove_domain_process.yml sd={{ item }}
```

Comment:
`sd={{ sd }}`
as you use loop_var sd

Reply:
but isn't the loop_var which is relevant is the one which called this task at clean_engine.py.
In this task we use item